docs/starting-from-scratch.md

```diff
@@ -10,323 +10,277 @@ single-spa allows you to build micro frontends that coexist and can each be writ
 2) Write code using a new framework, without rewriting your existing application
 3) [Lazy load](https://en.wikipedia.org/wiki/Lazy_loading) code for improved initial load time.
 
-Single-spa can be used with just about any build system or javascript framework, but this tutorial will focus on creating a web app with [Webpack](https://webpack.js.org/), [React](https://reactjs.org/), and [AngularJS](https://angularjs.org/). Our tutorial puts everything into a single code repository, but it is also possible to have [separate code repositories](separating-applications.md#option-3-dynamic-module-loading) for each of your applications.
+**Single-spa can be used with just about any build system or javascript framework**, but this tutorial will focus on creating a web app with [Webpack](https://webpack.js.org/), [React](https://reactjs.org/), and [AngularJS](https://angularjs.org/). Our tutorial puts everything into a single code repository, but it is also possible to have [separate code repositories](separating-applications.md#option-3-dynamic-module-loading) for each of your applications.
 
-You can find the [code for this tutorial here](https://github.com/alocke12992/single-spa-simple-example).
+For this tutorial we will be creating the following applications to showcase the power and usefulness of single-spa:
 
-If you'd like to learn how to use single-spa with Angular, Vue, or other frameworks, [checkout this example](https://github.com/CanopyTax/single-spa-examples). And if you'd rather use a different build system instead of webpack, check out [this example](https://github.com/me-12/single-spa-portal-example) Read more about [separating applications](separating-applications.md) using single-spa.
+1. home: a React app using [React Router](https://reacttraining.com/react-router/)
+2. navBar: a React app that always displays top-level navigation
+3. angularJS: an AngularJS app using [angular-ui-router](https://ui-router.github.io/ng1/)
 
-Be sure to read through the [single-spa docs](https://single-spa.js.org/), check the [single-spa github](https://github.com/CanopyTax/single-spa) and the [help section](https://single-spa.js.org/help.html) for more support.
+The complete code for this example is in the [`single-spa-simple-example`](https://github.com/alocke12992/single-spa-simple-example) repository.
 
-## Step One: Project Set up
+> __Note__
+>
+> We encourage you to read through all the [single-spa docs](/) to become familiar with the entire Single-spa setup. Visit the [single-spa Github](https://github.com/CanopyTax/single-spa), the [help section](https://single-spa.js.org/help.html), or our community [Slack](https://join.slack.com/t/single-spa/shared_invite/enQtMzIwMTcxNTU3ODQyLTM1Y2U1OWMzNTNjOWYyZDBlMDJhN2VkYzk3MDI2NzQ2Nzg0MzMzNjVhNWE2YjVhMTcxNjFkOWYzMjllMmUxMjk) channel for more support.
```

Comment:
Maybe `Single-spa` -> `single-spa`? You've been making that change pretty consistently in the rest of the PR, but not here

Reply:
Fixed.